docs/htdocs/info/docs/tools/vep/script/vep_cache.html

```diff
@@ -194,12 +194,12 @@ <h3 id="cache_content">Data in the cache</h3>
       </tr>
       <tr>
         <td><b>dbSNP</b></td>
-        <td>152</td>
+        <td>153</td>
```

Comment:
GRCh37 is using dbSNP 151

Reply:
Thanks, fixed. This is on the live site as well - it may be worth fixing